modules/querying-operator-status-after-installation.adoc

```diff
@@ -21,7 +21,52 @@ You can check Operator status at the end of an installation. Retrieve diagnostic
 $ oc get clusteroperators
 ----
 
-. If any Operators fail to become available, view Operator events:
+. Verify that all of the required certificate signing requests (CSRs) are approved. Some cluster Operators might not become available if there are pending CSRs.
+.. Check the status of the CSRs and ensure that you see a client and server request with the `Pending` or `Approved` status for each machine that you added to the cluster:
++
+[source,terminal]
+----
+$ oc get csr
+----
++
+.Example output
+[source,terminal]
+----
+NAME        AGE     REQUESTOR                                                                   CONDITION
+csr-8b2br   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Pending <1>
+csr-8vnps   15m     system:serviceaccount:openshift-machine-config-operator:node-bootstrapper   Pending
+csr-bfd72   5m26s   system:node:ip-10-0-50-126.us-east-2.compute.internal                       Pending <2>
+csr-c57lv   5m26s   system:node:ip-10-0-95-157.us-east-2.compute.internal                       Pending
+...
+----
+<1> A client request CSR.
+<2> A server request CSR.
++
+In this example, two machines are joining the cluster. You might see more approved CSRs in the list.
+
+.. If the CSRs were not approved, after all of the pending CSRs for the machines you added are in `Pending` status, approve the CSRs for your cluster machines:
++
+[NOTE]
+====
+Because the CSRs rotate automatically, approve your CSRs within an hour of adding the machines to the cluster. If you do not approve them within an hour, the certificates will rotate, and more than two certificates will be present for each node. You must approve all of these certificates. After you approve the initial CSRs, the subsequent node client CSRs are automatically approved by the cluster `kube-controller-manager`. You must implement a method of automatically approving the kubelet serving certificate requests.
```

Comment:
@staebler following detailed conversations with @sjenning and @kalexand-rh, I have created these two Bugzillas in relation to this comment:

* [Docs BZ 1914769](https://bugzilla.redhat.com/show_bug.cgi?id=1914769)
* [Engineering BZ 1914773](https://bugzilla.redhat.com/show_bug.cgi?id=1914773)

For the short term I have created [a new note in this PR](https://github.com/openshift/openshift-docs/pull/27823/files#diff-f0ffa2b36d6aeb62b9f56aef924dad3354ad2240bbb81066d3d7554a83299932R54-R57) so that the requirement and impact are clearer in the doc, at a high level. [The note in the original 'Approving the certificate signing requests for your machines' section](https://github.com/openshift/openshift-docs/pull/27823/files#diff-b87e23ae97d2c1cbe9acaeea182675205279379acf45a75caf2cc94ba04f0a34R99-R102) has also been updated with the same change in this PR.

For the longer term [BZ 1914773](https://bugzilla.redhat.com/show_bug.cgi?id=1914773) is a requests for our Engineering team to create and maintain an example script for OCP 4 that satisfies this requirement. We can then embed that script into a new procedure in the documentation that describes how to set this up in detail.

Please let me know if the note that I have included in the PR is sufficient as a short term solution. Thanks!

@jianlinliu can you please review this new note from a QE perspective? Thanks!

@kalexand-rh can you please peer review this new note from a docs perspective? Thanks!